### DISCUSSION TITLE
我从一个项目里build了个cmake的包出来,我该怎么处理?

### DISCUSSION THREAD

**Discussion by 95833** [xmake-io/xmake#2891] [Q&A]
![KT)BYTRQRUWTNIC(95Q{$UA](https://user-images.githubusercontent.com/21006005/193304548-1f1f42b6-b42c-4493-8ec0-36d9e5764300.png)

这玩意应该是cmake的包吧, 我在windows下用xmake.  这玩意该怎么处理成xmake包?
或许我该复制到项目中直接add_includedirs和add_linkdirs?

**Comment by waruqi** [comment]
add_includedirs/add_linkdirs 或者参考 https://xmake.io/#/zh-cn/package/local_3rd_source_library
